contents/architecture/RESTArchitecturalStyle.md

```diff
@@ -0,0 +1,226 @@
+# REST Architectural Style
```

Comment:
It'd be good to have links to other resources for students to refer to, just like the other book chapters